kube_controller_manager/assets/dashboards/overview.json

```diff
@@ -1,338 +1,911 @@
+						},
+						"layout": {
+							"x": 8,
+							"y": 0,
+							"width": 4,
+							"height": 4
+						}
+					}
+				]
+			},
+			"layout": {
+				"x": 0,
+				"y": 15,
+				"width": 12,
+				"height": 5,
+				"is_column_break": true
+			}
+		},
+		{
+			"id": 7885145587384818,
+			"definition": {
+				"title": "HTTP Requests",
+				"type": "group",
+				"background_color": "green",
+				"show_title": true,
+				"layout_type": "ordered",
+				"widgets": [
+					{
+						"id": 6389822199554693,
+						"definition": {
+							"title": "HTTP client requests by (method,code)",
+							"title_size": "16",
+							"title_align": "left",
+							"show_legend": true,
+							"legend_layout": "auto",
+							"legend_columns": [
+								"avg",
+								"min",
+								"max",
+								"value",
+								"sum"
+							],
+							"type": "timeseries",
+							"requests": [
+								{
+									"formulas": [
+										{
+											"formula": "query1"
+										}
+									],
+									"response_format": "timeseries",
+									"queries": [
+										{
+											"query": "sum:kube_controller_manager.client.http.requests{$cluster,$scope} by {method,code}.as_count()",
+											"data_source": "metrics",
+											"name": "query1"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "bars"
+								}
+							],
+							"custom_links": []
+						},
+						"layout": {
+							"x": 0,
+							"y": 0,
+							"width": 6,
+							"height": 4
+						}
+					}
+				]
+			},
+			"layout": {
+				"x": 0,
+				"y": 0,
+				"width": 6,
+				"height": 5
+			}
+		},
+		{
+			"id": 8277375896164218,
+			"definition": {
+				"title": "Rate Limits",
+				"type": "group",
+				"background_color": "purple",
+				"show_title": true,
+				"layout_type": "ordered",
+				"widgets": [
+					{
+						"id": 6648231289941672,
+						"definition": {
+							"title": "Rate limiter uses",
+							"title_size": "16",
+							"title_align": "left",
+							"show_legend": false,
+							"legend_size": "0",
+							"type": "timeseries",
+							"requests": [
+								{
+									"response_format": "timeseries",
+									"queries": [
+										{
+											"data_source": "metrics",
+											"name": "query1",
+											"query": "avg:kube_controller_manager.rate_limiter.use{$scope,$cluster} by {limiter}"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "bars"
+								}
+							],
+							"custom_links": []
+						},
+						"layout": {
+							"x": 0,
+							"y": 0,
+							"width": 6,
+							"height": 4
+						}
+					}
+				]
+			},
+			"layout": {
+				"x": 6,
+				"y": 0,
+				"width": 6,
+				"height": 5
+			}
+		},
+		{
+			"id": 56137128632033,
+			"definition": {
+				"title": "Go Metrics",
```

Comment:
for the kube_scheduler, the "GO metrics" also contains the thread numbers. 
maybe to have the same "look and feel" we could add it here.